src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -412,7 +412,11 @@ export default function DownloadDialog({
     return t('button.no-path-selected', 'No path selected')
   }
 
-  const isWebGame = gameInstallInfo?.game['name'] === 'web'
+  const isWebGame =
+    gameInstallInfo?.game['name'] === 'web' ||
+    gameInstallInfo?.game['name'] === 'webgl' ||
+    platformToInstall === 'Browser'
```

Comment:
webgl platform is not a web game. we should remove that line

Reply:
fixed